docs/changelog/91813.yaml

```diff
@@ -0,0 +1,5 @@
+pr: 91813
+summary: Skip remote clusters when performing up front privileges validation
+area: Transform
```

Comment:
I changed the release note to specifically say datafeeds because DFA doesn't currently support CCS.

```suggestion
summary: Skip remote clusters when performing up front privileges validation for datafeeds
area: Machine Learning
```

Reply:
I'll apply this suggestion once the release note file re-appears.

Reply:
Done.